packages/db/prisma/schema.prisma

```diff
@@ -25,3 +25,14 @@ model AuthKey {
 
   @@index([user_id])
 }
+
+model RefreshToken {
+  id           String   @id @unique @default(uuid())
+  createdAt    DateTime @default(now())
+  updatedAt    DateTime @updatedAt
+  userId       String
```

Comment:
According to how prisma works, this probably should just reference User

[1]

Reply:
Oh, okay, I didn't realise that's how this works. I was basing this off of the `AuthKey` model above, but I guess it's just defined in a weird way that the library expects. I'll correct this.